### PR TITLE
New Package: Stardock Fences version 4

### PR DESCRIPTION
**Pull request by gervasiocaj**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37841)

### CODE REVIEW

File: manifests/s/Stardock/Fences/4/Stardock.Fences.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS0.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: Stardock.Fences
+PackageVersion: "4"
```
Comment (ItzLevvie) [inline manifests/s/Stardock/Fences/4/Stardock.Fences.locale.en-US.yaml:5]:
```suggestion
PackageVersion: "4.03"
```

File: manifests/s/Stardock/Fences/4/Stardock.Fences.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS0.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Stardock.Fences
+PackageVersion: "4"
```
Comment (ItzLevvie) [inline manifests/s/Stardock/Fences/4/Stardock.Fences.installer.yaml:5]:
```suggestion
PackageVersion: "4.03"
```

File: manifests/s/Stardock/Fences/4/Stardock.Fences.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS0.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Stardock.Fences
+PackageVersion: "4"
```
Comment (ItzLevvie) [inline manifests/s/Stardock/Fences/4/Stardock.Fences.yaml:5]:
```suggestion
PackageVersion: "4.03"
```

Comment (ItzLevvie) [reply manifests/s/Stardock/Fences/4/Stardock.Fences.yaml:5]: To match Control Panel which prevents upgrade loops.

File: manifests/s/Stardock/Fences/4/Stardock.Fences.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/s/Stardock/Fences/4/Stardock.Fences.locale.en-US.yaml:5]: To match Control Panel which prevents upgrade loops.

File: manifests/s/Stardock/Fences/4/Stardock.Fences.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/s/Stardock/Fences/4/Stardock.Fences.installer.yaml:5]: To match Control Panel which prevents upgrade loops.

Comment (ItzLevvie) [reply manifests/s/Stardock/Fences/4/Stardock.Fences.installer.yaml:5]: You'll also need to update the version folder from `4` to `4.03`.

File: manifests/s/Stardock/Fences/4/Stardock.Fences.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/s/Stardock/Fences/4/Stardock.Fences.locale.en-US.yaml:5]: You'll also need to update the version folder from `4` to `4.03`.

File: manifests/s/Stardock/Fences/4/Stardock.Fences.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/s/Stardock/Fences/4/Stardock.Fences.yaml:5]: You'll also need to update the version folder from `4` to `4.03`.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by gervasiocaj** [comment]
Thank you for the feedback!

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Version folder needs to be updated to `4.03`

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**